### DISCUSSION TITLE
possibility of supporting multi-track m3u playlists with prev and next stream buttons?

### DISCUSSION THREAD

**Discussion by junguler** [muxinc/media-chrome#285] [Q&A] [answered]
is it possible to use a multi-track m3u playlist with this player? i'm using a js code i've [found](https://www.draketo.de/software/m3u-player.html) online inside my [repo](https://github.com/junguler/m3u-radio-music-playlists) to have these playable online but my knowledge of these stuff is very limited so i have to use the default audio player and it doesn't look good

i've been searching for a simple and good looking audio player with playlist support online for several days now but i haven't been able to find a good solution yet

here is an [example](https://junguler.github.io/m3u-radio-music-playlists/stuff/trance.html) of my ugly but working playlist buttons using the default audio player (i aligned the keys to firefox's audio player)

**Comment by luwes** [comment, marked as answer]
@junguler Media Chrome is used to build the UI controls of a media player and can work with different video engines that could support the functionality you like. See https://github.com/muxinc/media-chrome#compatible-players

Best to open an issue in a video engine's Github repo like hls.js or Shaka player.

---

If you are okay with playlist functionality loaded with multiple M3U8 files there is a project that can handle this and there is a PR open to add an example to Media Chrome https://github.com/muxinc/media-chrome/pull/247
See the `media-playlist` project here https://github.com/muxinc/media-playlist

**Reply by junguler** [reply]
thanks for your detailed answer, i really appreciate it
